odl/solvers/functional/functional.py

```diff
@@ -1450,9 +1420,9 @@ def point(self):
         return self.__point
 
     @property
-    def func_subgrad(self):
-        """The subgradient operator used to define the Bregman distance."""
-        return self.__func_subgrad
+    def subgrad(self):
+        """The subgradient used to define the Bregman distance."""
+        return self.__subgrad
 
     @property
     def initialized(self):
```

Comment:
Can you please also remove this property? I don't think it should be public.

Reply:
Just to log my own thinking: I left it because I though that one might be interested in checking which subgradient was used to define the distance.

Ok, will remove.

Reply:
If this is removed, how should the comment by @mehrhardt be handled?

More precisely: should I completely removed this function, or should I just remove the `@property` flag and rename it to `_subgrad`? If I should completely remove it, should I work with `self.__subgrad` in the rest of the functional?

Reply:
We're talking about the `initialized` property, which I think is a bit weird to have, since I would think that once the object has been constructed and initialized, i.e., `__init__` is over, it should be initialized. And it's also not necessary for the current implementation, it could easily be changed to the `... is None` check.

Reply:
Ah! Sorry, my bad. I was looking at the part of the code that was marked in colors :sweat_smile: